sdk/iot/provisioning/src/az_iot_provisioning_client.c

```diff
@@ -17,13 +17,13 @@ static const az_span str_dps_registrations_res
     = AZ_SPAN_LITERAL_FROM_STR("$dps/registrations/res/");
 
 // /registrations/
-AZ_INLINE az_span get_str_registrations()
+AZ_INLINE az_span _az_iot_provisioning_get_str_registrations()
 {
   return az_span_slice(str_dps_registrations_res, 4, 19);
 }
 
 // $dps/registrations/
-AZ_INLINE az_span get_str_dps_registrations()
+AZ_INLINE az_span _az_iot_provisioning_get_str_dps_registrations()
```

Comment:
macro for online function?

Reply:
I think inline is better than a macro. Let's postpone optimizations for later.
One thing that I will do is properly prefix all of these with `_`. If we decide to make changes, we can find them faster.